staging/src/k8s.io/client-go/examples/workqueue/main.go

```diff
@@ -116,30 +118,30 @@ func (c *Controller) handleErr(err error, key string) {
 }
 
 // Run begins watching and syncing.
-func (c *Controller) Run(workers int, stopCh chan struct{}) {
+func (c *Controller) Run(ctx context.Context, workers int) {
 	defer runtime.HandleCrash()
```

Comment:
Yes, we should. Let me do it now because you pointed it out.

Otherwise I would get to it when converting `client-go/examples` to contextual logging, because then `logcheck` will call it out automatically.